### PR TITLE
fixed #173 added checksum file to the result folder. It checks crc fo…

### LINKED ISSUE #173: COE: Clearly indicate when a result no longer is reproducible from a given MultiModel+CoSim config using a different icon for the R_ folder in the project browser
Clearly indicate when a result no longer is reproducible from a given MultiModel+CoSim config using a different icon for the R_ folder in the project browser
-  Suggestion: use checksum of MM and coe config. This can then be compared the current MM and coe config

### PR DESCRIPTION
**Pull request by lausdahl**
…r: mm, coe and output.csv

btw: do any if you know why debugging in electron no longer works. I can add breakpoints but they seem to set them self on a line of their choice and the context does not match when hit.

### CODE REVIEW

**Comment by ldcouto** [comment]
LGTM

I don't know why the app is not debugging. I'm experiencing the same. The sourcemaps are embedding and getting picked up so I'm not sure what the problem is.

I also get a ton of type errors when building the current master.

Someone please open an issue about this.

**Comment by CThuleHansen** [comment]
@ldcouto try deleting your typings folder and do gulp init gulp build

File: src/intocps-configurations/ResultConfig.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    //let res = { mm_config_crc: mmCrc, coe_config: coeCrc, result: resultCrc }
+    let ok = true;
+
+    let mmCrc = obj["mm_config_crc"];
+    if (mmCrc != null) {
+        let mmPath = Path.join(dir, "..", "..", "mm.json")
+        //console.debug("MM path: " + mmPath);
+        let crc = checksum(fs.readFileSync(mmPath).toString(), "md5", "hex");
+        //console.debug("crc: " + mmCrc + " == " + crc);
+        ok = ok && (crc == mmCrc);
```
Comment (CThuleHansen) [inline src/intocps-configurations/ResultConfig.ts]: ok = ok?

**Comment by CThuleHansen** [comment]
Looks fine by me...

**Comment by ldcouto** [comment]
@CThuleHansen that works, thanks.

Debugging is still broken for the co-sim configs, which are also broken in general (see #203). 

I do get an error when trying to create cosims, etc.:

```
zone-node.js:158 Uncaught Error: Command failed: git add -f "C:\Users\CoutoLD\projs\workspaces\into-app\tests\Multi-models\mm-new (70)\co-sim-62\coe.json"
fatal: Not a git repository (or any of the parent directories): .git
```

**Comment by CThuleHansen** [comment]
@ldcouto guess it is because the project is not a git repository? Then you need to turn traceability off

**Comment by ldcouto** [comment]
I guessed as much. But... 

I never chose to make the project a git repository (or not make it). If git is needed for app traceability, a repository needs to be created in the background when we create a project.

Anyway, even with traceability disabled, debugging and co-sim config is still broken: 

I tried to pause in the debuger a few times and it always seems to hit `patchTimer` in `zonde-node.js`. Could it be looping there?

**Comment by CThuleHansen** [comment]
@ldcouto  same error or different error?

**Comment by ldcouto** [comment]
It's still the thing in #203 

We should move this discussion elsewhere. We are polluting this PR like crazy

**Comment by CThuleHansen** [comment]
@ldcouto can you approev this PR?

**Comment by ldcouto** [comment]
Sorry. Thought I had.

**Pull request merged**